custom_gear_utils.py

```diff
@@ -0,0 +1,172 @@
+import os, os.path as op
+import subprocess as sp
+import sys
+from collections import OrderedDict
+import psutil
+
+import flywheel
+
+def Build_CPAC_Params(context):
```

Comment:
Make name generic
```suggestion
def Build_Params(context):
```

Reply:
RE: Remove _CPAC:
Generally, a good idea here.

In another gear, hcp-struct, we have three Build/Validate/Execute blocks that need to be named separately.

With only one block here, I can see better readability.